packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -217,54 +216,58 @@ export function connectAugur(
     if (loggedInAccountType === ACCOUNT_TYPES.TORUS) {
       preloadAccount(ACCOUNT_TYPES.TORUS);
     }
-
-    connect(
+    let provider = new JsonRpcProvider(env['ethereum-node'].http);
+    if (window.web3) {
+      provider = new Web3Provider(window.web3.currentProvider);
+    }
+    const sdk = await augurSdk.makeClient(
```

Comment:
You're in an `async` function, so you can just `await` here.

Reply:
Tom pointed out that they want this to happen in parallel with the other initialization as the app can init and then get an error where it puts up the error modal.

So this fine.